mediaapi/routing/download.go

```diff
@@ -61,6 +64,9 @@ type downloadRequest struct {
 	ThumbnailSize      types.ThumbnailSize
 	Logger             *log.Entry
 	DownloadFilename   string
+	forFederation      bool // whether we need to return a multipart/mixed response
```

Comment:
Flag is unclear. Is this implying that the download is over fed? Or the response will be sent via fed?

Reply:
That it's the response for an incoming federation request.